charts/internal/cilium/values.yaml

```diff
@@ -326,6 +326,12 @@ global:
     # protocols is the list of protocols to support
     protocols: tcp,udp
 
+  # Configure service load balancing
+  loadBalancer:
+    # serviceTopology enables K8s Topology Aware Hints -based service
+    # endpoints filtering
+    serviceTopology: false
```

Comment:
Topology aware routing is enabled per service via an annotation. Why do we require to enable it for cilium in addition?
In other words: Why do we not enable it per default when users anyway need to enable it per service?

Reply:
> In other words: Why do we not enable it per default when users anyway need to enable it per service?

Good point. Initially, I was trying to expose (make configurable) what cilium exposes as config option. This is a good question to the cilium folks as well. Why it is a config option on their side while it is a feature gate on kube-proxy side that will eventually be promoted and removed. 

So, do you want me to set this config to `true` and do not expose this setting? I think that makes sense.

Reply:
I asked your question to the cilium folks - ref https://github.com/cilium/cilium/issues/25283.

Reply:
Thanks, I now adapted.